src/commands/prepare.go

```diff
@@ -0,0 +1,206 @@
+package commands
+
+import (
+	"github.com/bwmarrin/discordgo"
+	"sandwich-delivery/src/config"
+	"sandwich-delivery/src/database"
+	"sandwich-delivery/src/models"
+	"strconv"
+	"time"
+)
+
+type PrepareOrderCommand struct{}
+
+func (c PrepareOrderCommand) getName() string {
+	return "prepareorder"
+}
+
+func (c PrepareOrderCommand) getCommandData() *discordgo.ApplicationCommand {
+	return &discordgo.ApplicationCommand{Name: c.getName(), Description: "Prepares the invite for the order so you can send it to the customer."}
```

Comment:
better description might be:
"Marks the order as out for delivery and sends you an invite to the server."